packages/web3-eth/src/schemas.ts

```diff
@@ -284,15 +293,33 @@ export const blockSchema = {
 		nonce: {
 			format: 'uint',
 		},
-		totalDifficulty: {
+		number: {
 			format: 'uint',
 		},
-		baseFeePerGas: {
-			format: 'uint',
+		parentBeaconBlockRoot: {
+			format: 'bytes32',
+		},
+		parentHash: {
+			format: 'bytes32',
+		},
+		receiptsRoot: {
+			format: 'bytes32',
+		},
+		sha3Uncles: {
+			format: 'bytes32',
 		},
 		size: {
 			format: 'uint',
 		},
+		stateRoot: {
+			format: 'bytes32',
+		},
+		timestamp: {
+			format: 'uint',
+		},
+		totalDifficulty: {
+			format: 'uint',
+		},
```

Comment:
these fields are already in current schema but this PR is only rearranging these to diff lines.